java-frontend/src/main/java/org/sonar/java/se/ExplodedGraph.java

```diff
@@ -133,6 +134,10 @@ public String toString() {
     public MethodYield selectedMethodYield(Node from) {
       return edges.containsKey(from) ? edges.get(from).yields.stream().findFirst().orElse(null) : null;
     }
+
+    Collection<Edge> edges() {
```

Comment:
Maybe that's just on me but I find it confusing to have the name "edges" on the node (parentEdges maybe ?)